python/redmonster/datamgr/io2.py

```diff
@@ -1221,6 +1254,11 @@ def merge_chi2(self):
             topdir = None
             print("'REDMONSTER_SPECTRO_REDUX' env variable not set.")
         try:
+            rmver = environ['REDMONSTER_VER']
```

Comment:
Both of these are only used (I think..) in my SDSS-specific routines, such as IO.  They are equivalent to `RUN1D` and `BOSS_SPECTRO_REDUX`.  It seems to me like DESI could take them or leave them.